config.ru

```diff
@@ -5,3 +5,4 @@
 require_relative 'config/environment'
 
 run Rails.application
+Rails.application.load_server
```

Comment:
I think this line is redundant.

Reply:
I compared it to Scholarsphere, and it doesn't have that line. 🤷

Reply:
Interesting. The update task recommended adding this. @EricDurante - do you have any thoughts here?

Reply:
Nm... I checked a `rails new` setup and it is as above with both lines, so leave this as-is.

Reply:
I'm not sure. If this is indeed incorrect or unnecessary, then it's a mistake in the rails generators/updater. With Rails 6.1, `rails new` generates a rack config file that looks like this.